cmd/deploy.go

```diff
@@ -147,7 +147,11 @@ func runDeployCmd(cmd *cobra.Command, args []string) error {
 	reportPodsStatus(ctx, kubeClient, namespace, sentryHelmContext)
 
 	if err != nil {
-		return errors.Wrap(err, "Helm installation validation failed")
+		fmt.Printf("\nHelm installation validation failed with error: %v\n", err.Error())
```

Comment:
Lets write something less self blaming

"Installation takes longer then expected, you can check the status using helm ls -n {{namespace}}"

Reply:
what if helm is not installed?